install_files/ansible-base/roles/validate/tasks/validate_tails_environment.yml

```diff
@@ -26,3 +26,54 @@
       The SecureDrop git repository should be cloned
       to `~/Persistent/securedrop`.
   with_items: "{{ tails_persistence_check_result.results }}"
+
+- name: Check for v3 client auth files
+  stat:
+    path: "/home/amnesia/Persistent/securedrop/install_files/ansible-base/{{ item }}"
+  register: v3_client_auth_files
+  with_items:
+    - app-ssh.auth_private
+    - app-journalist.auth_private
+    - mon-ssh.auth_private
+
+- name: Count the number of v3 client auth files
+  set_fact:
+    v3_client_auth_file_count: "{{ v3_client_auth_files.results | selectattr('stat.exists') | list | count }}"
+
+- name: Check for Tor v3 key file
+  stat:
+    path: "/home/amnesia/Persistent/securedrop/install_files/ansible-base/tor_v3_keys.json"
+  register: v3_tor_key
+
+- name: Confirm that v3 auth files are present for SSH-over-LAN configuration
+  assert:
+    that:
+      - v3_client_auth_file_count == "0" or v3_client_auth_file_count == "1"
+    msg: >-
+      There are multiple `.auth_private` files under
+      `~/Persistent/securedrop/install_files/ansible-base`, most likely from
+      a previous installation. Please remove any `.auth_private` files except
+      for `app-journalist.auth_private`, and retry the install command.
+  when: v3_onion_services == True and enable_ssh_over_tor == False
+
+- name: Confirm that v3 auth files are present for SSH-over-Tor configuration
+  assert:
+    that:
+      - v3_client_auth_file_count == "0" or v3_client_auth_file_count == "3"
+    msg: >-
+      Some v3 onion service client auth files are missing. Please add
+      all 3 `.auth_private` files under
+      `~/Persistent/securedrop/install_files/ansible-base`.
+  when: v3_onion_services == True and enable_ssh_over_tor == True
```

Comment:
consider a similar scenario for SSH over LAN:

1. User has a SecureDrop on 0.14.0 using SSH over LAN (has 0 .auth_private files)
2. User updates to 1.0.0, adds v3 onion service support (now has 1 .auth_private files)
3. User then decides to turn on SSH over Tor and disable SSH over LAN at some later date. The validation here will fail since they have only 1 `.private_auth` files in their admin workstation when they run the playbook. 

suggestion: we relax the validation here to allow 0, 1 or 3 .auth_private files

Reply:
Probably same deal, testing for SSH separately. So there should be be 0 or 2 ssh auth files.